scipy/io/matlab/mio.py

```diff
@@ -157,6 +157,47 @@ def loadmat(file_name, mdict=None, appendmat=True, **kwargs):
     >>> mat_contents['testdouble']
     array([[0.        , 0.78539816, 1.57079633, 2.35619449, 3.14159265,
             3.92699082, 4.71238898, 5.49778714, 6.28318531]])
+
+    By default SciPy reads MATLAB structures as structured NumPy arrays of
+    dtype object. This can be disabled by setting the optional arguement
```

Comment:
"dtype object" is a bit obscure.  How about "structured NumPy arrays, where the fields in the structures become fields in the structured NumPy array."

Reply:
OK, I changed the wording to:
>By default SciPy reads MATLAB structs as structured NumPy arrays where the
>dtypes are objects named after the MATLAB struct fields. This can be
>disabled by setting the optional argument `struct_as_record=False`.

IMO I think it's important to tell the user that the NumPy structured array fields will be *objects* which was not intuitive to me, I thought that a MATLAB struct field full of double arrays would be a equivalent to a named field in NumPy also of double arrays, but no.

Also I guess this raises the issue, that if *dtype object* is too obscure here, how will the user react when they see the MATLAB struct of strings, doubles and complex types become *objects* in NumPy? Perhaps there should be a cross reference to NumPy structured arrays or dtypes in see also? or elsewhere?